.codegen/service.py.tmpl

```diff
@@ -44,8 +44,12 @@ class {{.PascalName}}{{if eq "List" .PascalName}}Request{{end}}:{{if .Descriptio
 {{end}}
 {{- define "from_dict_type" -}}
 	{{- if not .Entity }}None
-	{{- else if .Entity.ArrayValue }}{{if or .Entity.ArrayValue.IsObject .Entity.ArrayValue.IsExternal}}_repeated(d, '{{.Name}}', {{.Entity.ArrayValue.PascalName}}){{else}}d.get('{{.Name}}', None){{end}}
-	{{- else if or .Entity.IsObject .Entity.IsExternal }}_from_dict(d, '{{.Name}}', {{.Entity.PascalName}})
+	{{- else if .Entity.ArrayValue }}
```

Comment:
can you do `{{template "type-nq" .Entity.ArrayValue}}`?

Reply:
Done, works for both cases below.